src/main/java/io/crate/testing/CrateTestServer.java

```diff
@@ -75,6 +76,7 @@
 
     public static final String DEFAULT_WORKING_DIR = System.getProperty("user.dir");
     public static final String CRATE_INSTANCES_DIR = "/parts";
+    private static final String TMP_DIR = System.getProperty("java.io.tmpdir");
```

Comment:
it would be nice to create a subfolder for the downloads. eg. crate.testing.downloads